lazyflow/utility/helpers.py

```diff
@@ -250,7 +250,45 @@ def __iter__(self):
         retSlice = [(self.toSlice(src,True),self.toSlice(trgt,True),self.toSlice(mask)) \
                     for src,trgt,mask in retRoi]
         return retSlice.__iter__()
- 
+
+
+def get_default_axisordering(shape):
+    """Given a data shape, return the default axis ordering
+
+    For data types that do not support axistags, we assume a default axis
+    ordering, given the shape, and implicitly the number of dimensions.
+
+    Args:
+        shape (tuple): Shape of the data
+
+    Returns:
+        str: String, each position represents one axis.
+    """
+    axisorders = {
+        2: 'yx',
+        3: 'zyx',
+        4: 'zyxc',
+        5: 'tzyxc'
+    }
```

Comment:
> At least we can control it from a single point now.
Amen to that. 

We want to use tzyxc internally, but how exactly we interpret the input is open. I just read some random big tiff i found on my hdd with vigra, it turned out as xyzc. Another one with colors was xyc, but maybe it's vigra default order? How would you check the order "in the wild"? Interestingly, if you try to read a 2D RGB image with vigra.readVolume, it just eats RAM and hangs. I wonder what causes that.